.travis.yml

```diff
@@ -17,3 +13,4 @@ before_script:
   - npm prune
 after_success:
   - npm run semantic-release
+  - npm run deploydocs
```

Comment:
So we should change this here too:
https://github.com/hoodiehq/pouchdb-hoodie-api/blob/master/.travis.yml#L24